### DISCUSSION TITLE
GTA V is not entering.

### DISCUSSION THREAD

**Discussion by EatingIting** [Whisky-App/Whisky#130] [Q&A]
<img width="800" alt="image" src="https://github.com/IsaacMarovitz/Whisky/assets/88502727/213e3aa5-ac49-452f-ad3a-58f760698037">

i ran the GTA V but this animation is infinity loop.

How can i play GTA V

**Comment by sussyboiiii** [comment]
Hello, 

same thing here, you can choose online or story mode in Steam and it wont get passed that screen (or it takes too long for my neves and a reasonable loading time).
Using GPT like on the applegamingwiki though started the game and also went into the splashscreen when selecting storymode in Steam which it doesn't seem to want to do with Whyski.

**Reply by sussyboiiii** [reply]
I want to add that if you run the bottle directly through GPT the game runs if a bit poorly compared to to crossover 22 on the M1 Pro atleast.
